src/sentry/api/endpoints/group_events.py

```diff
@@ -171,6 +175,17 @@ def _get_search_query_and_tags(self, request, group, environments=None):
         raw_query = request.GET.get('query')
 
         if raw_query:
+            # TODO(lb): remove once boolean search is fully functional
+            has_boolean_operators = features.has(
+                'organizations:boolean-search',
+                group.project.organization,
+                actor=request.user)
+
+            query_has_boolean_ops = [True for op in BOOLEAN_OPERATORS if op in raw_query]
```

Comment:
Could users have `AND` as part of a tag key or tag value?

Reply:
yeah this is problematic all around. I'm going to send it through the parser instead and check if a booleanSearch comes out instead.